### PR TITLE
Desktop dot release docs

### PR DESCRIPTION
**Pull request by amyblais**
Will add release dates when the releases are ready.

### CODE REVIEW

**Comment by devinbinnie** [comment]
@amyblais Going to add one more ticket, as it fixes an issue a few community members have been seeing on Linux: https://mattermost.atlassian.net/browse/MM-60233

**Review by cwarnermm — APPROVED** [review]

File: source/about/desktop-app-changelog.md [hunk 1]
```diff
@@ -8,6 +8,15 @@ This changelog summarizes updates to Mattermost desktop app releases for [Matter
 (release-v5-10)=
 ## Release v5.10
 
+- **v5.10.2, released **
```
Comment (devinbinnie) [inline source/about/desktop-app-changelog.md:11]: @amyblais We're missing MM-60233 from the list

Comment (amyblais) [reply source/about/desktop-app-changelog.md:11]: There is no release note on the PR description https://github.com/mattermost/desktop/pull/3157, but I'll add it to the changelog.

Comment (devinbinnie) [reply source/about/desktop-app-changelog.md:11]: Ahh sorry about that, I can add one.

Comment (amyblais) [reply source/about/desktop-app-changelog.md:11]: Thanks!

**Review by devinbinnie — APPROVED** [review]

**Pull request merged**